Mage/src/main/java/mage/abilities/dynamicvalue/common/SavedDamageValue.java

```diff
@@ -10,7 +10,8 @@
  */
 public enum SavedDamageValue implements DynamicValue {
     MANY("many"),
-    MUCH("much");
+    MUCH("much"),
+    DAMAGE("damage");
```

Comment:
No longer need to add this since the text isn't being used in the new card anyway